internal/app/referenceclient/impl.go

```diff
@@ -117,8 +119,24 @@ func (i *invoker) unary(
 	var trailers []*v1.Header
 	payloads := make([]*v1.ConformancePayload, 0, 1)
 
+	ctx = withWireCapture(ctx)
+
 	// Invoke the Unary call
 	resp, err := i.client.Unary(ctx, request)
+
+	var actualStatusCode int32
+	var connectErrorRaw *structpb.Struct
+	var actualTrailers []*v1.Header
+	fmt.Fprintf(os.Stderr, "Getting wire details in client impl for %s\n\n", req.TestName)
```

Comment:
Temp for testing - occasionally this and line 137 will print before setting the value in `wire_tracer`. It will be evident because a test will fail with status code 0.